pwa/components/common/Layout.tsx

```diff
@@ -17,7 +33,19 @@ const Layout = ({
 
   return (
     <QueryClientProvider client={queryClient}>
-      <Hydrate state={dehydratedState}>{children}</Hydrate>
+      <Hydrate state={dehydratedState}>
+        <AuthProvider>
+          <ThemeProvider theme={theme}>
+            <CssBaseline />
```

Comment:
I don't see breaking changes in the front, but before, this component had other components as children, but not now.
It's normal ?

Reply:
Yes, it's a self closing component, see: https://mui.com/material-ui/react-css-baseline/
There is another component for wrapping children: ScopedCssBaseline - https://mui.com/material-ui/react-css-baseline/#scoping-on-children